daffodil-cli/LICENSE-bin

```diff
@@ -232,8 +238,14 @@ subcomponents is subject to the terms and conditions of the following licenses.
     OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
     OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-  This product bundles source from 'Scala'. Specifically 'Utility.scala' and
-  'UniquenessCache.scala' The source is available under a BSD LICENSE.
+  This product bundles source from 'Scala', including the following files:
```

Comment:
Since this is the binary license, you'll want to point to the actual binary files and their locations.